pages/docs/frontend-modules/development.en-US.mdx

```diff
@@ -13,39 +15,120 @@ yarn
+<Callout type="info">
+  **`openmrs` tooling deep dive**
+
+`yarn start` is a shorthand for running the [openmrs CLI](https://github.com/openmrs/openmrs-esm-core/tree/main/packages/tooling/openmrs). More specifically, it's aliased to run `openmrs develop`, which creates a development build of the app shell and runs it against a given OpenMRS backend. It spins up a Webpack dev server on port 8080 and proxies requests to the default backend, which is hosted at https://dev3.openmrs.org. This dev server will by default run local copies of the frontend modules you specify and override their import map entries. All the other frontend modules run on that dev server should be the latest versions from the community dev3 server. With `openmrs develop`, you can also:
+
+- Specify an alternative backend server (other than the community dev3 server) and proxy requests to it. For example, to run the Vitals app and proxy requests to the O3 QA dev environment's backend, you can run:
+
+  ```sh
+  yarn start --sources packages/esm-patient-vitals-app --backend=https://test3.openmrs.org
+  ```
 
-- Specifying an alternative backend server (other than the community dev3 server) and proxying requests to it.
 - Specifying an alternative target server path (defaults to `/openmrs/spa`).
 - Specifying an alternative API URL (defaults to `openmrs`).
 - Specifying a custom import map.
 
+To learn more about the `develop` command, read through its [implementation](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/tooling/openmrs/src/commands/develop.ts).
+
+</Callout>
+
+## Using import map overrides
+
+O3 determines which frontend modules to load based on the [import map](/docs/core-concepts#import-map) provided to the app shell. The import map is a JSON object that maps module specifiers to URLs. You can find the import map for your O3 distro by navigating to `/openmrs/spa/importmap.json` in your browser. The [Devtools](https://github.com/openmrs/openmrs-esm-core/tree/main/packages/apps/esm-devtools-app) frotend modules provides an interface for overriding entries in your import map. You can leverage this when doing local development to load copies of your frontend modules from your local dev server. For example, if you fire up a [local development server running the app shell](docs/frontend-modules/development#running-the-app-shell-locally), it's possible to override import map entries for multiple frotend modules from multiple projects at the same time. This is useful when you're working on a feature that spans multiple frontend modules.
```

Comment:
This is nitpicking, but the import map actually tells us where to get the JS for a module. It's the routes registry that determines what we load. Maybe that's too much detail for these docs?